src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -790,7 +785,7 @@ protected TokenTaskResult doInBackground(String... urlItems) {
                 // Record account in the AccountManager service
                 try {
                     setAccount(result);
-                } catch (Exception exc) {
+                } catch (GeneralSecurityException | IOException exc) {
```

Comment:
Why do you need different catches and duplicated code if java supports multi-catch?